### ISSUE TITLE
New SwissBib: new OAI and metadata format changes

### ISSUE THREAD

**Issue by klokan** [klokantech/kartenportal-tickets#234]
Format changes
The most important information is still in the same fields, this shouldn't pose any bigger problems. If you have questions regarding some fields just ask. A general definition can be found here: http://www.swissbib.org/wiki/index.php?title=Swissbib_marc
Be aware that we changed the structure of 852/949, which you most likely use for linking to the source systems.
-- You have to take into account some changes in the swissbib-library-codes. The definition can be found: http://www.swissbib.org/wiki/index.php?title=Members_Talk:Library_codes
-- In a nutshell the codes for RERO, Alexandria and National library are changing, the rest stays the same.
-- Günter is preparing a dump for you in the coming weeks. We put some work in updating the addresses, which might be useful for you.

There is a new OAI interface available for testing (the final URL to be provided)

**Comment by klokan** [comment]
New OAI is now used on the new Kartenportal server.
We are using the testing server address.

Upgrade to the final OAI server address to be made - once it is known from SwissBib.

**Comment by klokan** [comment]
The new OAI interface for testing is not available anymore. We are getting "Could not connect to host". Upgrade to final OAI server address is urgent.

**Comment by klokan** [comment]
The OAI server runs now again - but the final address has been requested from SwissBib.

**Comment by klokan** [comment]
Final address updated. Closing.
